compiler/resolution/functionResolution.cpp

```diff
@@ -4296,6 +4297,7 @@ resolveCall(CallExpr* call)
      case PRIM_TUPLE_EXPAND:        resolveTupleExpand(call);           break;
      case PRIM_SET_MEMBER:          resolveSetMember(call);             break;
      case PRIM_MOVE:                resolveMove(call);                  break;
+     case PRIM_INITIALIZER_SET_FIELD: resolveFieldInit(call);           break;
```

Comment:
I will update alignment in a separate commit, I didn't want to muddy the change with non-functional operations.